pkg/cmd/attestation/verification/attestation.go

```diff
@@ -17,6 +17,7 @@ import (
 )
 
 var ErrUnrecognisedBundleExtension = errors.New("bundle file extension not supported, must be json or jsonl")
+var ErrEmptyJSONLFile = errors.New("provided jsonl file is empty")
```

Comment:
Might make sense to refer to this as a "bundle file" b/c of the flag name?

Reply:
Good call